### ISSUE TITLE
Maximum score - does not update existing submissions; Problem weight doesn't work?

### ISSUE THREAD

**Issue by lukasc-ubc** [mitodl/edx-sga#138]
I initially assumed that I can leave the maximum score to say 10, and put the Problem Weight to 1.0.  The idea being that it would contribute 1 point to the Unit.  (I have other multiple choice questions each worth 1 point, and I want the ubcpi to be 1 point).

However, that wasn't the case. And I discovered this part-way through my class.

I changed the Maximum Score to 1, instead of 10.  But that didn't update existing submissions.  Nor did changing the problem weight.  I couldn't figure out how to make any changes to existing submissions.

**Comment by pdpinch** [comment]
Have you tried adjusting the grading policy for the course? http://edx.readthedocs.org/projects/edx-partner-course-staff/en/latest/grading/index.html

There is also a known issue with grade weights, described in #95. Are you having a similar problem?

**Comment by lukasc-ubc** [comment]
Yes, thank you.  Looks like part of the problem is the same as #95.  The other part being that you can't modify the score mid-way, while students are submitting their submissions..
